### PR TITLE
Implementation Views

### PR DESCRIPTION
**Pull request by rica213**
Hello, 

In this PR, here are what I have done: 
- Add images to my assets
- Implement the actions in my post controller
- Change title in the layout
- Create templates (partial for user, post, and comment)
- Create views corresponding to users
- Create views corresponding to posts
- Refactor tests based on the new views
- Define basic layouts for the views
- Update readme file

That's it 😆😆😆
Happy reviewing!

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -1 +1,17 @@
...
+  <article class='post'>
+    <h2> <%= @post.title %> by <%= @user.name %></h2>
+    <section class='post_card__stat'>
+      <p> Comments: <%= @post.comments_count %> </p>
+      <p> Likes: <%= @post.likes_count %>
+      </section>
+      <p> <%= @post.text %> </p>
+      <% unless @post.recent_comments.empty? %>
+        <section class='comment_list'>
+          <% @post.recent_comments.each do |comment| %>
```
Comment (Kamba56) [inline app/views/posts/show.html.erb:11]: - [ ] When displaying a single post, all comments of that post should be displayed. It is only in the index page that the five recent comments should be displayed.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @rica213, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job using partials in views👏👏👏
- [x] Great job adding views to your blog.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (rica213) [reply app/views/posts/show.html.erb]: Thank you for pointing that out 😁 It is done 👍🏽

**Review by TadesseDev — APPROVED** [review]
### Hi @rica213 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**